src/behat/ConfigurationPage.php

```diff
@@ -133,6 +138,11 @@ public function setProperties($properties)
                 case 'radio':
                     $radio = $this->context->assertFind('css', $propertyLocator . '[value="' . $value . '"]');
                     try {
+                        // if the parent is td tag then doesn't throw exception
```

Comment:
I don't see the connection between this comment and the if.
What do you want to test exactly ?